Parsers/ASim/Watchlists/ASimDisabledParsers.json

```diff
@@ -11,12 +11,12 @@
     },
       "resources": [
           {
-          "name": "[concat(parameters('workspaceName'), '/Microsoft.SecurityInsights/ASimDisabledParser')]",
+          "name": "[concat(parameters('workspaceName'), '/Microsoft.SecurityInsights/ASimDisabledParsers')]",
```

Comment:
Since this doesn't cause any interruption in functionality, and a change might cause problems for users who already deployed the watchlist if deploying again - I prefer not to change the resource name

Reply:
There are 13 Asim parsers that use _GetWatchlist('ASimDisabledParsers') so providing a watchlist template with a different name as guidance seems confusing to me. People who might have deployed the watchlist now might not even know the parsers are looking for a different watchlist.

Reply:
You are right! Sorry!

Reply:
you are right! sorry!